### PR TITLE
SRVCOM-1288 + SRVCOM-1664: Clean up attributes, prereqs, metadata

### PR DESCRIPTION
**Pull request by abrennan89**
Jiras: https://issues.redhat.com/browse/SRVCOM-1288 + https://issues.redhat.com/browse/SRVCOM-1664

Applies for OCP 4.6+

Direct preview: https://deploy-preview-41648--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-apiserversource

### CODE REVIEW

File: modules/apiserversource-yaml.adoc [hunk 1]
```diff
@@ -1,74 +1,23 @@
 // Module included in the following assemblies:
 //
-// serverless/knative_eventing/serverless-listing-event-sources.adoc
+// * serverless/develop/serverless-apiserversource.adoc
 
 :_content-type: PROCEDURE
 [id="apiserversource-yaml_context"]
-= Using the API server source with the YAML method
+= Creating an API server source by using YAML files
```
Comment (abrennan89) [inline modules/apiserversource-yaml.adoc:7]: Note to self: update topicmap so left nav matches this

**Review by aliok — APPROVED** [review]
/lgtm

I don't see anything inaccurate or strange.

File: serverless/develop/serverless-apiserversource.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 :_content-type: ASSEMBLY
 [id="serverless-apiserversource"]
-= Using the API server source
+= Creating an API server source
```
Comment (jrangelramos) [inline serverless/develop/serverless-apiserversource.adoc:3]: dumb question... should the title on left menu follow the title here as well?

**Review by jrangelramos — COMMENTED** [review]
looks good to me (w/ just a single note / question)

**Pull request merged**